src/styles/common/empty-network.css

```diff
@@ -8,6 +8,10 @@
     width: 5em;
     padding: 20px 0px 0px 0px;
   }
+
+  & .empty-network-message {
```

Comment:
Do you still need this css if you are using the component?

Reply:
I removed the css from the enrichment.css file since we are using the component.  This part in empty-network.css is to add spacing between the image and the message. I like the view better with the space, but I can remove this portion to. 

![image](https://user-images.githubusercontent.com/38890251/44273277-02336f80-a20d-11e8-8bbf-18cc97954667.png)
 VS
![image](https://user-images.githubusercontent.com/38890251/44273291-11b2b880-a20d-11e8-995e-531f652f030f.png)

Reply:
Ok that makes sense